descriptions/ntdelayexecution.md

```diff
@@ -15,6 +15,10 @@ Despite the name, `NtAlertThreadByThreadId` is unrelated to alertable sleeps and
 
 # Related Win32 API
  - [`SleepEx`](https://learn.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-sleepex)
+ - [`Sleep`](https://learn.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-sleep)
+
+# Documented by
+* Wojciech Dudek
```

Comment:
This might seem as if the whole document was authored by you. I'm not sure what's the best solution here. At this point, I think it's best to keep these documents without the "Documented by" section. The authors can be queried via GitHub, and I may add a more explicit list of authors on the website itself in the future.

It was also briefly discussed here: https://github.com/m417z/ntdoc/pull/3

@cyber-wojtek @diversenok what do you think?

Reply:
Yeah I was thinking the same

Reply:
I was planning to eventually phase out most of these "Documented by" sections by rewriting the corresponding pages. I don't see much value in keeping them for functions/types already extensively documented by Microsoft, but I also don't feel like I have a right to remove them on other pages without reworking the content,